build.sbt

```diff
@@ -1287,6 +1298,65 @@ lazy val runtime = (project in file("engine/runtime"))
   .dependsOn(`docs-generator`)
   .dependsOn(testkit % Test)
 
+lazy val `runtime-instrument-id-execution` = (project in file("engine/runtime-instrument-id-execution"))
+  .settings(
+    inConfig(Compile)(truffleRunOptionsSettings),
+    instrumentationSettings
+  )
+  .dependsOn(runtime)
+
+lazy val `runtime-instrument-repl-debugger` = (project in file("engine/runtime-instrument-repl-debugger"))
+  .settings(
+    inConfig(Compile)(truffleRunOptionsSettings),
+    instrumentationSettings
+  )
+  .dependsOn(runtime)
+
+lazy val `runtime-instrument-runtime-server` = (project in file("engine/runtime-instrument-runtime-server"))
+  .settings(
+    inConfig(Compile)(truffleRunOptionsSettings),
+    instrumentationSettings
+  )
+  .dependsOn(runtime)
+
+lazy val `runtime-with-instruments`  = (project in file("engine/runtime-instrument"))
```

Comment:
I'd suggest making the project directory and project variable name consistent, unless we have a solid reason not to.

Reply:
Thanks! That was on my list yesterday but I seemed to have forgotten about it